common/common.go

```diff
@@ -290,8 +290,13 @@ func RunBuild(verbose bool, dir, imageName, dockerfile string, buildArgs []strin
 }
 
 func dockerVersionCheck() error {
-	out, err := exec.Command("docker", "version", "--format", "{{.Server.Version}}").Output()
-	if err != nil {
+	out, err := exec.Command("docker", "version", "--format", "{{.Server.Version}}").Output() //executes the "docker version" command and gets the output and error code
+	if err != nil {                                                                           //returns exit status 1 err if docker is not running
+		errString := err.Error() //converts the err to string
+		//fmt.Printf(errString)
+		if errString == "exit status 1" {
+			return fmt.Errorf("Cannot connect to the docker daemon. Is the docker daemon running?: %v", err)
+		}
```

Comment:
I'm not sure we need this after https://github.com/fnproject/cli/pull/441/files ?